evap/evaluation/management/commands/refresh_results_cache.py

```diff
@@ -18,9 +17,6 @@ class Command(BaseCommand):
     requires_migrations_checks = True
 
     def handle(self, *args, **options):
-        self.stdout.write("Clearing results cache...")
-        caches["results"].clear()
```

Comment:
I don't understand why we wouldn't clear it anymore (even if we are overwriting all keys anyways)

Reply:
Ah, before the caching was lazy and we only let it update by requesting the template. Should we rename `warm_up_template_cache` to `update_template_cache` then?